src/components/EditableComponent.tsx

```diff
@@ -93,6 +93,10 @@ class EditableComponent<P extends MappedComponentProperties, S extends Container
         if (appliedCssClassNames)
             sProps.className += appliedCssClassNames + ' ';
 
+        if (this.props.containerProps && this.props.containerProps.className){
+            sProps.className = sProps.className + ' ' + this.props.containerProps.className;
+        }
+    
```

Comment:
You might need to run the linter if there any available on the package json scripts.

Reply:
You can simplify it with:

```typescript
if (this.props?.containerProps?.className){
    sProps.className += ' ' + this.props.containerProps.className;
}

```